src/components/layout.scss

```
@@ -484,6 +484,10 @@ $dropdown-item-active-background-color: $primary;
       background-color: white;
 
       &.character-container {
+        .columns:not(:last-child) {
+          margin-bottom: -0.75rem;
```

`columns`はデフォルトで下方向に-0.75remしてくれないんですね･･･。
`hr`を使っているのがデザインフレームワーク的に良くない感じな気がしてきました。
デザインフレームワークの上から独自UIを載せるの大変ですね。。。

こっちも意図をちょっとコメント書いとくのどうでしょう 👀 
（ちょっと謎コメントになっちゃいましたが･･･）
```suggestion
          margin-bottom: -0.75rem; // columnsは下方向のmargin調整がないので、hrの辻褄を合わせに必要
```